### PR TITLE
Initial Solution Pack Merge

### PR DESCRIPTION
**Pull request by fortijames**
This is the first release of the solution pack for review. This currently has 8 examples but we are building more and will create pull requests as they are completed.

### CODE REVIEW

**Comment by cs-rohit-khune** [comment]
@fortijames Please make sure to set the version to 1.0.0 and commit your solution pack code to the release/1.0.0 branch instead of the develop branch.

File: info.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
+{
+    "name": "fortiManagerZTPFlow-FeatureExamples",
+    "version": "1.0.9",
```
Comment (cs-rohit-khune) [inline info.json:3]: @fortijames Please make sure to set the version to 1.0.0

File: info.json [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    "fsrMinCompatibility": "7.4.0",
+    "type": "solutionpack",
+    "local": true,
+    "label": "FortiManager ZTP Flow - Feature Examples",
+    "dependencies": [
+        {
```
Comment (cs-rohit-khune) [inline info.json:9]: Make sure to add solution pack dependency such as "SOAR Framework"

File: info.json [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+            "name": "fortiManagerZTPFlow",
+            "type": "solutionpack",
+            "label": "FortiManager ZTP Flow",
+            "minVersion": null
+        }
+    ],
+    "prerequisite": null,
+    "publisher": "Community",
```
Comment (cs-rohit-khune) [inline info.json:17]: Please ensure that the value of the "publisher" field is set to "Fortinet CSE".

File: info.json [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    "description": "Feature focused solution examples using the FortiManager ZTP Flow Solution Pack. Create device models in your FMG and apply example solutions to see in action what would happen before you apply solutions to your production gear.",
+    "help": "https://github.com/fortinet-fortisoar/solution-pack-fortimanager-ztp-feature-examples",
```
Comment (cs-rohit-khune) [inline info.json:19]: Please provide a link to README.md on the release/1.0.0 branch. i.e

https://github.com/fortinet-fortisoar/solution-pack-fortimanager-ztp-feature-examples/blob/release/1.0.0/README.md

File: info.json [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+    "category": [
+        "Compliance and Reporting",
+        "DevOps and Digital Operations",
+        "Centralized Security Management",
+        "Utilities"
+    ],
+    "supportInfo": "https://github.com/fortinet-fortisoar/solution-pack-fortimanager-ztp-feature-examples/issues",
```
Comment (cs-rohit-khune) [inline info.json:26]: Please leave this field blank

File: README.md [hunk 6]
```diff
@@ -1 +1,13 @@
-# solution-template-repository
\ No newline at end of file
+# Release Information
```
Comment (cs-rohit-khune) [inline README.md:1]: Please create documentation using the standard format provided in https://github.com/fortinet-fortisoar/how-tos/tree/main/documenting

**Review by cs-rohit-khune — CHANGES_REQUESTED** [review]

**Comment by fortijames** [comment]
Closing this PR for now as we figure out the paths to make incremental changes. Sometimes I might make multiple changes in one PR that would elevate the version beyond 1.0.x. Is having a per version branch required?

**Pull request closed without merging**